#GO 3: Initial Article (#2054)
ppy/osu-wiki: wiki/Tournaments/#GO/3/en.md

---
tags:

> - '#Game\_Osu Tourney 3'

Reviewer: don't need to use backslashes in this tags list

- '#GO 3'
- '#GO3'
- Game\_Osu Tourney 3
- GO 3
- GO3
---
# #Game\_Osu Tourney 3

![#GO 3 Logo](img/logo.png)

> The **#Game\_Osu Tourney 3** (**#GO 3**) is a [Logitech](https://www.logitech.com/ "Logitech website") sponsored worldwide one-on-one osu! tournament hosted by ![][flag_FR] [0xy](/users/9239673), ![][flag_FR] [Kaanibaru](/users/3117824) and supported by ![][flag_FR] [Sardoche](/users/4383702). It is the third installment of the #Game\_Osu Tourney.

Reviewer: ```suggestion
The **#Game\_Osu Tourney 3** (**#GO 3**) is a [Logitech](https://www.logitech.com/ "Logitech website") sponsored worldwide one-on-one osu! tournament hosted by ![][flag_FR] [0xy](/users/9239673) and ![][flag_FR] [Kaanibaru](/users/3117824), and supported by ![][flag_FR] [Sardoche](/users/4383702). It is the third installment of the #Game\_Osu Tourney.
```

## Tournament Schedule

| Event | Timestamp |
| --: | :-- |
| Registration Phase | 2019-02-16/2019-03-04 |
| Live Drawings | 2019-03-04 (20:00 UTC+0) |
| Qualifiers Stage | 2019-03-09/2019-03-10 |
| Round of 64 | 2019-03-16/2019-03-17 |
| Round of 32 | 2019-03-23/2019-03-24 |
| Round of 16 | 2019-03-30/2019-03-31 |
| Quarterfinals | 2019-04-06/2019-04-07 |
| Semifinals | 2019-04-13/2019-04-14 |
| Finals | 2019-04-20/2019-04-21 |
| Grand Finals | 2019-04-21 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 1000€, unique profile badge |
| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 500€ |
| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 250€ |
| *4th place* | 150€ |
| *5th place* | 100€ |

> *Prizes may change overtime. Such as addition of Logitech branded hardware (includes mice, headsets, keyboards, etc).*

Reviewer 2: ```suggestion
*Prizes may change over time such as an addition of Logitech branded hardware (e.g. mice, headsets, keyboards, etc).*
```

| Statistician | ![][flag_FR] [FayeurS 5](/users/6177263) |
| Developers | ![][flag_FR] [Loazo](/users/3925586), ![][flag_SE] [AntoN](/users/2538562), ![][flag_FR] [0xy](/users/9239673) |
| Wiki | ![][flag_ID] [Naruk](/users/7100002) |

## Links

- [Discussion thread](/community/forums/topics/869989)
- [#GO Discord server](https://discord.gg/trnFByC)
- [#GO Youtube channel](https://www.youtube.com/channel/UCwIJ0ymAahAASp0_KaZ_1jA)
- [Website](https://www.gameosu.com/)
- [Player registration form](https://www.gameosu.com/sign_up)
- Livestream
  - [#Game\_Osu (EN)](https://www.twitch.tv/gameosu)

>   - [Sardoche's Twitch channel](https://www.twitch.tv/sardoche)

Reviewer: what is this used for? doesn't seem to be explained anywhere

Author: I believe the host told me the Finals and Grand Finals will be streamed on Sardoche's. He's a massive FR Twitch streamer too, that may attract more viewers to the tournament.

Reviewer: can write that in parenthesis after this link then

  - [#Game\_Osu (FR)](https://www.twitch.tv/gameosu_fr)

-----------------------

## Participants

*The participants will be revealed after the registration phase closes.*

-----------------------

## Mappools

*The first mappools (for Qualifiers Stage) will be revealed on 2019-03-04 at 20:00 UTC+0.*

------------------------------------------------------------------------

## Ruleset

### General Rules

1. The #Game\_Osu is a one-on-one osu! tournament supported by Logitech.

> 2. The Qualifier will be in single-elimination.

Reviewer 2: ```suggestion
2. The Qualifiers will be in single-elimination.
```

3. In the Quarterfinals and the Semifinals, you need to win 6 beatmaps to win a match. (Best-of-11)
4. In the **Finals and Grand Finals**, you need to win 7 beatmaps to win a match. (Best-of-13)
   - The Grand Finals may occur a bracket reset if the player coming from the Losers bracket, wins the first match.
5. Referees will be holding host during the matches. Players will state their bans and picks in the MP chat, and the referees will select the maps.
6. The referees host multiple lobbies and tournaments at the same time and will not always know every map or their details. Please use the map's full name or terms such as `[mod][map number]` (using the `mappool` on the website tab as reference) to reduce possible confusion. Asking the referee to select maps that as examples; describe its background or reference will reduce your chances of getting the map you wish for.
   - For FreeMod picks, each player needs to pick at least one mod from: Hidden/HardRock/Easy/Flashlight. Multiple mods are allowed. These same mods are allowed to be used on the Tiebreaker.

### Stage Instructions

#### Qualifiers

> 1. Qualifier will be played in **groups of 4.** Each group will have a Top seed and one lower seeded player of each category.

Reviewer 2: ```suggestion
1. Qualifiers will be played in **groups of 4.** Each group will have a Top seed and one lower seeded player of each category.
```

2. If a player fails to join the lobby past 10 minutes of his scheduled match, we will declare the match a forfeit.
3. There will be no warmups, so make sure you are prepared before your matches!
4. Each player will `!roll`. The highest roll decides the pick and ban order (first to ban will be the last to pick).
5. In case of a tie, the Tiebreaker will be played. Losers in the Winners bracket will go to the Losers bracket.
6. If any participants experienced a disconnection in the middle of play from Bancho, a rematch of that same map may be played at the referee's discretion. The players are given 5 minutes after a disconnection to contact their referee via osu! or Discord before being counted a loss.
7. Rescheduling is allowed, however we ask of you to contact your opponent by yourself and then send us a proof that both of you are having an aggrement to reschedule. You can do so by posting a message on the `#rescheduling` text-channel on [#GO Discord server](https://discord.gg/trnFByC) with both you and your opponent acknowledging it. The deadline for rescheduling is on the Thursday 7PM (UTC) of the upcoming week of your match.

### Behaviour Rules

> 1. Every [osu! Rules](/wiki/Rules) applies.

Reviewer 2: ```suggestion
1. All [osu! Rules](/wiki/Rules) apply.
```

2. *Toxic* behaviour during matches will be highly reprimanded and may result in an **auto-loss, kick, or blacklist**.
3. We will not tolerate any ounce of **cheating or attempts to gain an unfair advantage over the other players.**

> 4. Please note that **only** the [#Game\_Osu](https://www.twitch.tv/gameosu) and [Sardoche's](https://www.twitch.tv/sardoche) Twitch channels are allowed to stream the tournament. **We do not allow restreams** unless given permission and will ask for **takedowns** in case of **unauthorized uploads.** Players are allowed to stream their matches.

Reviewer: missing one of the twitch streams that are linked above

> 4. Please note that **only** the [#Game\_Osu (EN)](https://www.twitch.tv/gameosu), [#Game\_Osu (FR)](https://www.twitch.tv/gameosu_fr), and [Sardoche's](https://www.twitch.tv/sardoche) Twitch channels are allowed to stream the tournament. **We do not allow restreams** unless given permission and will ask for **takedowns** in case of **unauthorized uploads.**However, players are allowed to stream their matches.

Reviewer: ```suggestion
4. Please note that **only** the [#Game\_Osu (EN)](https://www.twitch.tv/gameosu), [#Game\_Osu (FR)](https://www.twitch.tv/gameosu_fr), and [Sardoche's](https://www.twitch.tv/sardoche) Twitch channels are allowed to stream the tournament. **We do not allow restreams** unless given permission and will ask for **takedowns** in case of **unauthorized uploads.** However, players are allowed to stream their matches.
```
